blog_post/dbt_blog_post.md

```diff
@@ -0,0 +1,163 @@
+# Enforcing data access policies in dbt with PACE
+
+When we released our first alpha version of PACE a couple of months ago, it was in the form of a
+(Spring Boot) server application. Integrating with various data processing platforms, and pulling
+metadata from a number of data catalogs, PACE started our journey towards open-source data access
+policy management. Users can define data policies (column masks/transforms, row filters) in
+a yaml format, and enforce them on their processing platform by using the PACE cli (command line
+interface), or alternatively the REST or gRPC API directly. Currently, PACE applies the policies by
+generating platform-specific SQL statements that create views on the targeted platform, which in
+their turn leverage native access controls to implement the desired access policy. Got a table with
+with sensitive (customer) data? Let PACE create a (privacy-)safe view for it!
+
+[//]: # (Todo: an illustration here?)
```

Comment:
Good one. Maybe a screenshot of BQ? Or Snowflake? Or both? To highlight the fact that PACE is platform agnostic? 

Or even an outline of the workflow?